frontend/js/behaviors/layeredImageViewer/layeredImageViewer.js

```diff
@@ -0,0 +1,1339 @@
+
+      // Description populated by the alt text.
+      // This is supplemental and read second to the label (skipped in some cicrumstances)
+      // ariaDescription due to become standard in ARIA 1.3, for now describedBy has better compatability
+      // described-by also allows multiple ID's
+      this.viewer.canvas.setAttribute(
+        'aria-describedby',
+        `layered-image-viewer-${this.id}-images-0`
+      );
+
+      // Activate default overlays
+      const changeEvent = new Event('change');
+
+      this.overlays.default.forEach((index)=> {
+        this.overlays.items[index].checkboxEl.checked = true;
+        this.overlays.items[index].checkboxEl.dispatchEvent(changeEvent);
+      });
+
+      // Add resizeObserver
+      this._setViewerResizeObserver();
+
+      // Force interaction setter function to fire
+      this.interaction = false;
+    });
+  }
+
+  /**
+   * Destroy Viewer
+   * @public
+   * @static
+   * @method
+   * @param {HTMLElement} viewerEl - Element which was used to initialise the viewer to be destroyed
+   */
+  static destroy(viewerEl) {
+    // Qualifying elements should have a layeredImageViewer stored inside
+    const instance = viewerEl.layeredImageViewer
+
+    if (!instance) return;
+
+    // Stop observers
+    instance.viewerResizeObserver.disconnect();
+
+    // Destroy OSD and remove elements added by this class
+    instance.viewer.destroy();
+    viewerEl.querySelector('.o-layered-image-viewer__osd').remove();
+
+    // Cull events added outside the scope of the viewer
+    ['click', 'touchend'].forEach((event) => {
+      document.body.removeEventListener(
+        event,
+        instance.boundHandleExternalClick,
+        true
+      );
+    });
+
+    delete viewerEl.layeredImageViewer;
+  }
+
+  /**
+   * Transform string into kebab-case
+   * @public
+   * @static
+   * @method
+   * @param {String} string - String to transform
+   * @returns {String} - Kebab-case string
+   */
+  static toKebabCase(string) {
+    return string.toLowerCase().replace(' ', '-');
+  }
+
+  /**
+   * Transform string into camelCase
+   * @public
+   * @static
+   * @method
+   * @param {String} string - String to transform
+   * @returns {String} - camelCase string
+   */
+  static toCamelCase(string) {
+    return string
+      .toLowerCase()
+      .replace(/[^a-zA-Z0-9]+(.)/g, (m, chr) => chr.toUpperCase());
+  }
+
+  /**
+   * Transform an array of strings into a sentenace-like string
+   * @public
+   * @static
+   * @method
+   * @param {Array} list - List of strings to serialize
+   * @returns {String} - String of joined array items
+   */
+  static stringifyList(list) {
+    let string = '';
+    if (list.length > 1) {
+      if (list.length === 2) {
+        string = list.join(' and ');
+      } else {
+        // With an Oxford comma
+        string = `${list.slice(0, -1).join(', ')}, and ${list.slice(-1)}`;
+      }
+    } else {
+      string = list.toString();
+    }
+    return string;
+  }
```

Comment:
It might be useful to put the above three functions into a helpers/utilities file.

Reply:
We don't currently have a place for shared functions like. I'll create a ticket for this change.